File: El Mamoun/Lv 1/Problem 48.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
Problem 48: Write a program to read a LoanAmount and ask you how many months you need to settle the loan, then 
calculate the monthly installment amount
*/

namespace Problem_48
{
    internal class Program
    {

        static int ReadPostiveNumber(string Message)
        {

            int Number = 0;
            do
            {
                Console.WriteLine(Message);
                Number = Convert.ToInt32(Console.ReadLine());

            } while (Number < 0);

            return Number;
        }

        static float CalculateMonthlyInstallmentAmount(int LoanAmount,int HowManyMonth)
        {
            return (float)LoanAmount /(float) HowManyMonth;
        }

        static void Main(string[] args)
        {

            int LoanAmount = ReadPostiveNumber("Please Enter a Loan Amount?: ");
            int  HowManyMonth = ReadPostiveNumber("Please Enter a How many Months?: ");

            Console.WriteLine(CalculateMonthlyInstallmentAmount(LoanAmount, HowManyMonth));
            Console.ReadKey();

        }
    }
}

File: El Mamoun/Lv 1/Problem 48.cpp
#include<iostream>
using namespace std;

/*
Problem 48: Write a program to read a LoanAmount and ask you how many months you need to settle the loan, then
calculate the monthly installment amount
*/

int ReadPostiveNumber(string message)
{
	int number{ 0 };
	do
	{
		cout << message;
		cin >> number;

	} while (number < 0);

	return number;
}

float CalMonthlyInstallmentAmount(float LoanAmount, float HowManyMonth)
{
	return (float)LoanAmount / HowManyMonth;
}

int main()
{
	float LoanAmount = ReadPostiveNumber("Please Enter Loan Amount? ");
	float HowManyMonth = ReadPostiveNumber("Please Enter How many Month ? ");

	cout << "Number of Month is= " << CalMonthlyInstallmentAmount(LoanAmount, HowManyMonth);

	return 0;
}